app/javascript/components/widgets/weather/weather-effect.jsx

```diff
@@ -248,6 +269,68 @@ const Thunder = () => {
     </>
   );
 };
+// #endregion
+
+// cloud starts here
+const CloudMotionFrames = keyframes`
+   0% {
+    margin-left:-1000px;
+  }
+  100% {
+    margin-left:100%;
+  }
+`;
+
+const MoveUpDown = keyframes`
+0%{
+    transform: translateY(0);
+  }
+  30%{
+    transform: translateY(40px);
+  }
+  60% {
+    transform: translateY(-100px); 
+  }
+  100% {
+    transform: translateY(0);
+  }
+`;
+
+const CloudImage = css`
+  opacity: 60%;
+  background: white;
+  width: 200px;
+  height: 150px;
+  border-radius: 0 0 50px 50px;
+  background: radial-gradient(35px 30px, white 98%, transparent 100%) 20% 30% /70px
+      60px,
+    radial-gradient(50px 45px, white 98%, transparent 100%) 50% 50% /100px 90px,
+    radial-gradient(50px, white 98%, transparent 100%) 100% 100%/100px 100px,
+    radial-gradient(40px, white 98%, transparent 100%) 0 100% /80px 80px,
+    linear-gradient(white, white) bottom/100% 40px;
+  background-repeat: no-repeat;
+`;
+
+const MovingCloud = styled.div`
+  bottom: 100%;
+  position: absolute;
+  top: 120px;
+  animation: ${CloudMotionFrames} linear, ${MoveUpDown} linear;
+  ${CloudImage};
+`;
```

Comment:
Nice composition of the CloudImage with MovingCloud. I saw you passed two arguments to the animation field - can you pass as many animation arguments as you want?

Reply:
@tucker-weed  Yes , you can.